upgrade/1.4.4/README.md

```diff
@@ -0,0 +1,375 @@
+
+   ```text
+   INFO: Querying CFS configurations for the following NCNs: x3000c0s5b0n0, ...
+   INFO: Found configuration "management-csm-1.4.0" for component x3000c0s5b0n0
+   ...
+   INFO: Updating existing layer with repo path /vcs/cray/csm-config-management.git and playbook site.yml
+   INFO: Property "commit" of layer with repo path /vcs/cray/csm-config-management.git and playbook site.yml updated ...
+   INFO: Property "name" of layer with repo path /vcs/cray/csm-config-management.git and playbook site.yml updated ...
+   INFO: No layer with repo path /vcs/cray/csm-config-management.git and playbook ncn-initrd.yml found.
+   INFO: Adding a layer with repo path /vcs/cray/csm-config-management.git and playbook ncn-initrd.yml to the end.
+   INFO: Successfully saved CFS configuration "management-csm-1.4.0-backup-20230918T205149"
+   INFO: Successfully saved CFS configuration "management-csm-1.4.0"
+   INFO: Successfully saved 1 changed CFS configuration(s) to CFS.
+   INFO: Updated 9 CFS components.
+   INFO: Waiting for 9 component(s) to finish configuration
+   INFO: Summary of number of components in each status: pending: 9
+   INFO: Waiting for 9 pending component(s)
+   INFO: Sleeping for 30 seconds before checking status of 9 pending component(s).
+   ...
+   INFO: Sleeping for 30 seconds before checking status of 9 pending component(s).
+   INFO: 9 pending components transitioned to status configured: x3000c0s5b0n0, ...
+   INFO: Finished waiting for 9 component(s) to finish configuration.
+   INFO: Summary of number of components in each status: configured: 9
+   ====> Completed update of CFS configuration(s)
+   ====> Cleaning up install dependencies
+   ```
+
+   When configuration of all components is successful, the summary line will show all components
+   with status "configured".
+
+### Update NCN images
+
+NCN images must be rebuilt at this time in order to acquire an important Kernel panic mitigation. The mitigation entails
+a new Kernel and networking drivers for SP4 images \(Kubernetes\), as well as the blacklisting of the QLogic RDMA driver
+for SP4 and SP3 (all NCNs).
+
+Despite rebuilding both Kubernetes and Storage CEPH images, **only Kubernetes nodes will embark on a rolling rebuilt**.
+Storage CEPH nodes will receive an in-place modification, and do not need to be rebuilt at this time.
+
+#### Warnings
+
+***IMPORTANT*** This minor version bump has an unprecedented rolling rebuild. This is a friendly reminder that any 
+system administration data living on masters and workers **will be wiped** during the rebuild. Administrators are
+advised to take backups of their local, site files.
+
+> Examples:
+> - ` ~/.config/sat/sat`
+> - `/etc/motd`
+> - `/etc/sudoers`
+> - `/home`
+> - `/root/.ssh/config`
+
+
+#### Image Customization
+
+1. Follow option 1, 2, or 3 in [Stage 0.3](../Stage_0_Prerequisites.md#stage-03---update-management-node-cfs-configuration-and-customize-worker-node-image).  
```

Comment:
I think here they should only be following option 2 or 3. If they are going to upgrade other products (option 1) then we should direct them at the beginning of this procedure to not do the patch, and instead do a full IUF run upgrading products and CSM which is necessary because of the kernel fix. I don't think they would have to follow the patch procedure if ultimately they are going to do a CSM upgrade with IUF anyway.

Reply:
Also, I think a small description should be added for which option they should pick.

Reply:
I think I addressed this, please let me know what you think.

Reply:
Marking as resolved, any further resolution can be done via https://github.com/Cray-HPE/docs-csm/pull/4647#discussion_r1473366174